index.html

```diff
@@ -1,29 +1,227 @@
+        Eden needed
+        a new home. So, I've placed her in this universe, with some more fleshed out details, like a full name (Eden
+        Valdez), new hero name (LIGHTSTRIKE), and new position in The Guild!<br>
+        <button type="button">CHECK OUT THE CHANGES HERE!</button>
+      </p>
+    </div>
+    <!--SHOWCASES-->
+    <div class="showcase">
+      <!--CHARACTER SHOWCASE-->
+      <h2>FEATURED CHARACTERS SHOWCASE</h2>
+      <div class="showcase1">
 
-    <!-- We are using a CSS Reset file to standardize browser default css -->
-    <link rel="stylesheet" href="reset.css" />
+        <div>
+          <img src="images/MitziProfile.jpg" alt="An image of the character" width="200">
 
-    <!-- This is where all of our awesome CSS code goes -->
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <!--
+          <h3>"MITZI"</h3>
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+          <p>A deep sea Saphretean, thrust into a position of immense power because of a plan gone wrong and a relic of
+            godlike power, Mitzi has a very chaotic life. Luckily, her and her new pirate crew seem to be able to handle
+            it. Somewhat. <br>
+            <button type="button">VIEW CHARACTER</button>
+          </p>
+        </div>
+        <div>
+          <img src="images/Lighstrike.PNG" alt="An image of the character" width="200">
+          <h3>"LIGHSTRIKE"</h3>
 
-    -->
-  </body>
-</html>
+          <p>Eden Valdez, aka Lighstrike, used to be a hero. That was before her home dimension was ripped to shreads.
+            Having
+            been saved by a Glitch and placed in another world, she has been working as an assassin ever since, while
+            trying to find out if her old friends are still out there somewhere.<br>
+            <button type="button">VIEW CHARACTER</button>
+          </p>
+        </div>
+        <div>
+          <img src="images/VProfile.png" alt="An image of the character" width="200">
+          <h3>GUEST CHARACTER: "V"</h3>
+
+          <p>Created by Nicebur. <br><br>
+            V had a life before this. Before the darkness that clouds their vision, the memories barely clung to, and
+            the
+            hatred that burns slowly at their soul. The Glitch attempted to save them before, but are they even possible
+            to save now? <br>
+            <button type="button">VIEW CHARACTER</button>
+          </p>
+        </div>
+        <div>
+          <img src="images/WatcherZero.PNG" alt="An image of the character" width="200">
+          <h3>"WATCHER ZERO"</h3>
+
+          <p>PROTOCOL: ELIMINATE.<br> Watcher Zero, like the other Watchers, used to be a force to
+            keep the universe stable, but they now have a new prime directive: to hunt down Nyx,
+            eliminate her, and return the universe to their ideal state of order.<br>
+            <button type="button">VIEW CHARACTER</button>
+          </p>
+        </div>
```

Comment:
This section would be better marked up using a `<ul>` because it is a list of characters, as opposed to using `<div>s`

```html
<ul class="character-list">
  <li>
    <div>
     ... Character one ...
    </div>
  </li>
  <li>
    <div>
     ... Character two ...
    </div>
  </li>
</ul>
```